etc/dehydrated/hook.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
A TODO as you refine this: given how many HTTP calls we have to make in this script, we might want to consider setting `set -e` here to fail fast or have a trap on the error so that we get some nice noisy logging or problems.